### DISCUSSION TITLE
Bagaimana mengaktifkan port khusus di Elastic Beanstalk?

### DISCUSSION THREAD

**Discussion by misskecupbung** [awsugid/forum#5] [Q&A]
Halo mau bertanya untuk mengaktifkan port khusus di elastic Bean stalk ada khusus atau bisa dr security group saja ya?

**Comment by misskecupbung** [comment]
Bisa buat security group, inbound rules-nya dispesifikan utk port app/service/security group lain yg mau ada akses komunikasi dg beanstalknya pak. Sama seperti kalau kita buat ec2 saja, cuma ini di beanstalk

**Comment by vixus7891** [comment]
berikut contoh settingan port di Beanstalknya : 
![Setting Port Beanstalk](https://github.com/awsugid/forum/assets/147033632/ea0fed06-fecb-4ef8-a2e6-a92f56d1aae4)
